pom.xml

```diff
@@ -240,7 +240,7 @@
                         <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-21:latest</image>
                     </from>
                     <to>
-                        <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/local/lfp-pay-web:latest</image>
+                        <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/local/penalty-payment-web:latest</image>
```

Comment:
Yeah same as other comments, let's leave this for follow up config change ticket once the pipelines are renamed. Would've been good to have this sorted by now but lead times and resources...